### DISCUSSION TITLE
Images pasted from clipboard are not embedding in notebook file

### DISCUSSION THREAD

**Discussion by beckerben** [microsoft/vscode-jupyter#14408] [Questions and Answers]
I swear when I used to paste images from the clipboard into a markdown cell, the image would be embedded behind the scenes instead of stored with the file.  This has stopped happening now.  I have the "Ipynb › Paste Images As Attachments: Enabled".  What am I missing?  Did this break with an update to VSCode or the jupyter extension?

Version: 1.82.3 (system setup)
Commit: fdb98833154679dbaa7af67a5a29fe19e55c2b73
Name: Jupyter

Id: ms-toolsai.jupyter
Description: Jupyter notebook support, interactive programming and computing that supports Intellisense, debugging and more.
Version: 2023.8.1002501831
Publisher: Microsoft
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter

**Comment by DonJayamanne** [comment]
@amunger @Yoyokrazy  /cc

**Comment by amunger** [comment]
still works for me in the latest insiders on WIndows. What OS are you on?

**Reply by beckerben** [reply]
I tested on Win11 & Win10.  It works as long as you start a new workbook and don't save the jupyter notebook.  Once you save the notebook, your pastes of images from the clipboard will no longer be treated as inline attachments.

**Reply by amunger** [reply]
ah, thanks, now I can repro. created https://github.com/microsoft/vscode/issues/194809
